config/packages/app_event_admin.yaml

```diff
@@ -4,3 +4,38 @@ sulu_admin:
             routes:
                 list: 'app.get_events'
                 detail: 'app.get_event'
+    field_type_options:
+        selection:
+            event_selection:
+                default_type: 'list_overlay'
+                resource_key: 'events'
+                view:
+                    name: 'app.event.add_form'
+                    result_to_view:
+                        id: 'id'
+                types:
+                    list_overlay:
+                        adapter: 'table'
+                        list_key: 'events'
+                        display_properties:
+                            - 'name'
+                        icon: 'su-calendar'
```

Comment:
Available icons are documented in the [Sulu Javascript Docs](https://jsdocs.sulu.io/latest/#icon).